README.md

```diff
@@ -18,103 +18,187 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
+* Select desired API Levels (select all >= 23).
+* Click apply to download and install. 
+
+Once you have completed all of the required installs and setup, you should have the following environment variables set:
+```
+JAVA_HOME
+ANDROID_HOME
+ANDROID_SDK_HOME
+PATH
+``` 
+> Note: For additonal help setting these environment variables, refer to the  [Troubleshooting](#troubleshooting) section below.
+
+### 5. Add Android platform and plug-ins
+
+Add the Android platform as the target for your mobile app. Then install plug-ins to provide access to device-level features.
+
+```
+$ cd mobile
+$ cordova platform add android
+
+# ensure that everything has been installed correctly
+$ cordova requirements
+
+$ cordova plugin add cordova-plugin-camera
+$ cordova plugin add cordova-plugin-file-transfer
+```
+
+### 6. Setup your Android phone
+
+In order to run the mobile app on your phone, you will need to enable `developer options` and `web debugging`. 
+
+> Note: Please refer to documentation on your specific phone to set these options.
+
+Attach your Android phone to your computer.
+
+Install [Android File Transfer](https://www.android.com/filetransfer/) on your computer to enable file transfer between your computer and phone. 
+
+### 7. Build and run the mobile app
+
+```
+$ cd mobile
+$ cordova build android
+$ cordova run android
+```
+
+At this point, the app named `CalorieCounter` should come up on your phone. Press the `Capture Image` button to transfer to your phone camera. Take a photo of a food item and press `OK`. Analsis data will then be displayed.
+
+
+# Sample Output
   
-  Apache 2.0
+<img src="images/output1.jpg" width="250">  <img src="images/output2.jpg" width="250">
   
+# Links
+
+* [Watson Node.js SDK](https://github.com/watson-developer-cloud/node-sdk)
+
+# Troubleshooting
+
+* `cordova run android` error: Failure [INSTALL_FAILED_UPDATE_INCOMPATIBLE]
+
+> The `Calorie Counter` app is already installed on your phone and incompatible with the version you are now trying to run. Uninstall the current version and try again.
+
+* `cordova run android` error: No target specified and no devices found, deploying to emulator
+
+> Ensure that your phone is plugged into your computer and you can access it from the Android File Transfer utility (see Step #6 above).
+
+* How to determine proper values for environment variables:
+
+Open `Android Studio` and navigate to `File` -> `Project Structure` -> `SDK 
+Location`. This location value will serve as the base for your environement variables. For example, if the location is `/users/joe/Android/sdk`, then:
+
+```
+export ANDROID_HOME=/users/joe/Android/sdk
+export ANDROID_SDK_HOME=/users/joe/Android/sdk/platforms/android-<api-level> 
+export PATH=${PATH}:/users/joe/Android/sdk/platform-tools:/users/joe/Android/sdk/tools
+```
+
+# License
   
+[Apache 2.0](LICENSE)
+
+# Privacy Notice
+
+If using the Deploy to Bluemix button some metrics are tracked, the following
```

Comment:
this section is only applicable if we're using the d2b deployment tracker, which i don't think we are, unless you plan to do that, remove it

Reply:
@stevemart d2b deployment tracker is required since we are pushing server code to Bluemix